### ISSUE TITLE
FormLabel prop not working

### ISSUE THREAD

**Issue by aerfio** [SAP/fundamental-react#275]
`required` prop of FormLabel doesn't work, nothing happens if we add this.

Using example from https://sap.github.io/fundamental-react/forms in CRA
```jsx
<FormSet>
    <FormItem>
        <FormLabel htmlFor='input-3' required>
            Password
        </FormLabel>
        <FormInput type='password' id='input-3'
            placeholder='Field placeholder text' />
    </FormItem>
</FormSet>
```
That is what it gives me
![image](https://user-images.githubusercontent.com/17271979/51599553-29027a00-1f00-11e9-8d5d-2521a145b6d8.png)
That what we want to get:
![image](https://user-images.githubusercontent.com/17271979/51599617-5d763600-1f00-11e9-8e64-561678196a6b.png)

**Comment by CodesOfRa** [comment]
I think the ```is-required``` css class has been removed from fiori-fundamentals .  I can't find it in form.scss .

**Comment by chrismanciero** [comment]
Looking at the history of form.scss there was this change https://github.com/SAP/fundamental/commit/ebc719881d80834b6281319846725fb70c0c522c#diff-d34e54e5522d4f6a9720495b676d5c5a 
that removed is-required and added aria-required=true

**Comment by chrismanciero** [comment]
After speaking with @ChristianKienle, we found out that the FormLabel component needs to append the * after the label for required fields. Will create a PR and reference this issue

**Comment by CodesOfRa** [comment]
@aerfio this was fixed in 0.1.0 release.
